dundalk-institute-of-technology-harvard.csl

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" and="text" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Dundalk Institute of Technology Harvard</title>
+    <title-short>DkIT Harvard</title-short>
+    <id>http://www.zotero.org/styles/dundalk-institute-of-technology-harvard</id>
+    <link href="http://www.zotero.org/styles/dundalk-institute-of-technology-harvard" rel="self"/>
+    <link href="https://www.dkit.ie/library/support/guide-harvard-referencing" rel="documentation"/>
+    <contributor>
+      <name>Claire Fox</name>
+      <uri>http://www.mendeley.com/profiles/claire-fox1/</uri>
+      <email>claire.fox@dkit.ie</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Dundalk Institute of Technology Harvard Referencing Style</summary>
+    <updated>2016-02-09T15:32:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with=". " name-as-sort-order="all"/>
+      <label form="short" prefix=", " text-case="lowercase"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text variable="title" font-style="normal"/>
+  </macro>
+  <macro name="author">
+    <names variable="author" display="block" suffix=".">
```

Comment:
are you using all the `display="block"` with a particular goal? This will likely look weird in a bibliography.